File: Selenium/Activities/Activity7.java
package ibm.fst_m1_selenium;

import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity7 {
	public static void main(String[] args) {

        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        Actions builder = new Actions(driver);

        driver.get("https://v1.training-support.net/selenium/drag-drop");
        System.out.println("Home page title: " + driver.getTitle());

        WebElement football = driver.findElement(By.id("draggable"));
        WebElement dropzone1 = driver.findElement(By.id("droppable"));
        WebElement dropzone2 = driver.findElement(By.id("dropzone2"));

        builder.clickAndHold(football).moveToElement(dropzone1).pause(2000).release().build().perform();
        String dropzone1Verify = dropzone1.findElement(By.tagName("p")).getText();
        if(dropzone1Verify.equals("Dropped!")) {
            System.out.println("Ball was dropped in dropzone 1");
        }

        builder.dragAndDrop(football, dropzone2).build().perform();
        String dropzone2Verify = dropzone2.findElement(By.tagName("p")).getText();
        if(dropzone2Verify.equals("Dropped!")) {
            System.out.println("Ball was dropped in dropzone 2");
        }

             driver.close();
    }

}

File: Selenium/Activities/activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains

with webdriver.Firefox() as driver:
    driver.get("https://v1.training-support.net/selenium/drag-drop")
    print("Title is: ",driver.title)
    action = ActionChains(driver)

    Ball=driver.find_element(By.XPATH,"//img[@id='draggable']")
    Drop1=driver.find_element(By.XPATH,"//div[@id='droppable' and text()='Dropzone 1']")
    Drop2=driver.find_element(By.XPATH,"//div[@id='dropzone2' and text()='Dropzone 2']")
    action.drag_and_drop(Ball,Drop1).perform()
    print("Enter drop 1 ", driver.find_element(By.XPATH,"//div[@id='droppable' and text()='Dropzone 1']/p").text)
    action.drag_and_drop(Ball,Drop2).perform()
    print("Enter drop 2 ", driver.find_element(By.XPATH,"//div[@id='dropzone2' and text()='Dropzone 2']/p").text)